gravitee-kubernetes-client/src/test/java/io/gravitee/kubernetes/client/KubernetesUnitTest.java

```diff
@@ -40,7 +40,7 @@ public void before() {
 
         // Set kubeconfig system property to use for creating a
         buildKubernetesConfig();
-        kubernetesClient = new KubernetesClientV1Impl(vertx);
+        kubernetesClient = new KubernetesClientV1Impl();
```

Comment:
Can you have a look at the method `after()` which deals with the vertx instance? Now that the vertx instance is created by the kubernetesClient itself, I'm wondering if it still relevant?